scripts/nuget-debug-helpers.ps1

```diff
@@ -142,10 +144,21 @@ Function Add-NuGetToCLI {
 
     ## Copy the xplat artifacts
 
-    Get-ChildItem $nugetXplatArtifactsPath -Filter NuGet*.dll |
+    Write-Debug "Artifacts: $nugetXplatArtifactsPath"
+    Get-ChildItem -Recurse $nugetXplatArtifactsPath -Filter NuGet*.dll |
         Foreach-Object {
-            $new_position = "$($sdk_path)\$($_.BaseName )$($_.Extension )"
-
+            $currDir = $_.Directory.BaseName
+            Write-Debug "Parent $currDir" 
+            if ($locFolders -contains $currDir)
+            {
+                $new_position = "$($sdk_path)\$($currDir)\$($_.BaseName )$($_.Extension )"
+            }
+            else
+            {
+                $new_position = "$($sdk_path)\$($_.BaseName )$($_.Extension )"
+            }
+
+            Write-Debug "Moving $($_.FullName) to - $($new_position)"
```

Comment:
Isn't this line kind of overlaps with next one? Should we delete line 162?

Reply:
I want to keep both lines, one is for Informational purposes, the other line I'm adding is for debug purposes.

Reply:
But don't both of them always print?

Reply:
Only in Debug mode, i.e., when you set `$DebugPreference = "continue"` or `$DebugPreference = "stop"`. See https://ss64.com/ps/write-debug.html